conf/pxe_cluster/ubuntu-uefi-server.seed

```diff
@@ -30,7 +30,7 @@ d-i mirror/http/proxy string
 d-i clock-setup/utc-auto boolean true
 d-i time/zone string America/Denver
 d-i clock-setup/ntp boolean true
-d-i clock-setup/ntp-server string 192.168.0.1 
+d-i clock-setup/ntp-server string 192.168.0.1
```

Comment:
If we are setting ntp here, then we don't need it in the post.sh

192.168.0.1 should be changed to the {{.Machine.url}}